src/web/tests/test_services.py

```diff
@@ -58,3 +70,36 @@ def test_updates_event_instead_of_creating_new_one(self):
         assert event.name == "Intro to Dagger"
         assert event.description == "Super cool intro to Dagger CI/CD!"
         assert event.external_id == MockMeetupEventScraper.EXTERNAL_ID
+
+    def test_manually_applied_tags_are_not_overriden(self):
```

Comment:
This is great to check that Tags aren't getting overridden. Can you add a test that simply checks that Tags are added to an Event that doesn't already have Tags? That seems to not be working for me.